gateway/loaders/zookeeper/zookeeper.go

```diff
@@ -269,6 +271,9 @@ func (z *ZookeeperTargetLoader) WatchConfiguration(targetChan chan<- *connection
 			z.config.Log.Info().Msgf("[ZK] Successfuly set data watch for: %s", event.Path)
 		}
 	}
+Exit:
+	fmt.Println("Zookeeper connection closed")
```

Comment:
We may want to retry establishing the zk connection, the POD may have been moved/restarted.